crane/views/v2.py

```diff
@@ -60,10 +64,31 @@ def name_redirect(relative_path):
     base_url = repository.get_path_for_repo(name_component)
     if not base_url.endswith('/'):
         base_url += '/'
+
+    if 'manifests' in path_component:
```

Comment:
this logic is not correct, it should not make 302 redirects for docker pull by digest for schema2. I need to update this